### PR TITLE
Code Cleanup - MissingPluginException / Exception

### PR DESCRIPTION
**Pull request by pirouet**
Ensure files that use MissingPluginException or Exception have the appropriate use statement set

### CODE REVIEW

**Comment by pirouet** [comment]
Travis-CI failing in MemcachedEngineTest.php. My PR does not interfere with this file so I do not see it being related

**Comment by pirouet** [comment]
Can be seen passing on forked travis at:

https://travis-ci.org/pirouet/cakephp/builds/57284617

File: tests/TestCase/Error/ExceptionRendererTest.php [hunk 1]
```diff
@@ -583,21 +584,21 @@ public static function exceptionProvider()
...
-                new \RuntimeException('another boom'),
+                new RuntimeException('another boom'),
                 [
                     '/Internal Error/'
                 ],
                 500
             ],
             [
-                new \Cake\Core\Exception\Exception('base class'),
+                new CakeException('base class'),
```
Comment (jadb) [inline tests/TestCase/Error/ExceptionRendererTest.php]: Why was this changed to `CakeException`?

Comment (pirouet) [inline tests/TestCase/Error/ExceptionRendererTest.php]: Hi,

The PHP Exception is being used in this file also so to avoid confusion I renamed the Exception inside Cake namespace to have the CakeException alias to signify this is a Cake exception class

Comment (jadb) [inline tests/TestCase/Error/ExceptionRendererTest.php]: Ok, thanks - oversight on my part :)

Comment (pirouet) [inline tests/TestCase/Error/ExceptionRendererTest.php]: No problem :)
